aten/src/ATen/native/Convolution.cpp

```diff
@@ -239,6 +242,19 @@ auto ConvParams::use_nnpack(const at::Tensor& input) const -> bool {
   return false;
 }
 
+auto ConvParams::use_xnnpack(
+    const at::Tensor& input,
+    const at::Tensor& weight,
+    const at::Tensor& bias) const -> bool {
+// #if defined(C10_MOBILE)
```

Comment:
Shouldn't leave this comment in.

Reply:
No just trying to get CI coverage on non-mobile builds to fix potential issues.